src/lib/ext-activitylog.js

```diff
@@ -4,9 +4,9 @@ class Model {
   constructor() {
     this.logs = [];
     this.filter = {
-      id: [],
-      viewType: [],
-      type: [],
+      id: new Set([]),
```

Comment:
`new Set()` is sufficient. No need to pass an empty array. Let's fix this in the whole patch.